modules/apps/29-fee/keeper/keeper.go

```diff
@@ -241,6 +241,24 @@ func (k Keeper) DeleteForwardRelayerAddress(ctx sdk.Context, packetID channeltyp
 	store.Delete(key)
 }
 
+// GetDistributionAddress retrieves the fee distribution address stored in state given the provided channel identifier and relayer address
```

Comment:
This is a nit, but should we add these functions next to the ones to get and set the counterparty address?

Reply:
Yep, done! I've moved to above counterparty getter/setter funcs. And also re-ordered the rpc definitions in proto so `RegisterPayee` comes before `RegisterCounterpartyPayee`